README.md

```diff
@@ -8,131 +8,88 @@
-* [Chip](docs/view/Chip/README.md)
-* [Dialog](docs/view/Dialog/README.md)
-* [Horizontal Nav](docs/view/HorizontalNav/README.md)
-* [Floating Action Button](docs/view/FloatingActionButton/README.md)
-* [Interactive Star Rating](docs/view/InteractiveStarRating/README.md)
-* [Map Markers](docs/view/Maps/README.md)
-* [Nav Bar](docs/view/NavBar/README.md)
-* [Nudger](docs/view/Nudger/README.md)
-* [Panel](docs/view/Panel/README.md)
-* [RadioButton](docs/view/RadioButton/README.md)
-* [Overlay](docs/view/Overlay/README.md)
-* [Rating](docs/view/Rating/README.md)
-* [Slider](docs/view/Slider/README.md)
-* [Snackbar](docs/view/Snackbar/README.md)
-* [Spinner](docs/view/Spinner/README.md)
-* [Star Rating](docs/view/StarRating/README.md)
-* [Switch](docs/view/Switch/README.md)
-* [Text](docs/view/Text/README.md)
-* [Text Field](docs/view/TextField/README.md)
-* [Text Spans](docs/view/TextSpans/README.md)
-* [Toast](docs/view/Toast/README.md)
-
-## Usage
-
-### Radii
-
-The Backpack radii tokens are available as [dimension resource](Backpack/src/main/res/values/backpack.radii.xml).
-
-The supported tokens are
-
-+ `bpkBorderRadiusSm`
-+ `bpkBorderRadiusPill`
-
-### Elevation
-
-The Backpack elevation tokens are available as [dimension resource](Backpack/src/main/res/values/backpack.elevation.xml).
-
-The supported tokens are
-
-+ `bpkElevationXs`
-+ `bpkElevationSm`
-+ `bpkElevationBase`
-+ `bpkElevationLg`
-+ `bpkElevationXl`
-
-### Text Styles
-
-The Backpack text styles are available as [style resources](Backpack/src/main/res/values/backpack.text.xml).
-
-The supported styles are
-
-+ `bpkTextXs`
-+ `bpkTextXsEmphasized`
-+ `bpkTextSm`
-+ `bpkTextSmEmphasized`
-+ `bpkTextBase`
-+ `bpkTextBaseEmphasized`
-+ `bpkTextLg`
-+ `bpkTextLgEmphasized`
-+ `bpkTextXl`
-+ `bpkTextXlEmphasized`
-+ `bpkTextXl`
-+ `bpkTextXlEmphasized`
-+ `bpkTextXxl`
-+ `bpkTextXxlEmphasized`
-
-### Color
-```xml
-<TextView
-  android:text="This is Backpack Blue 500!"
-  android:textColor="@color/bpkSkyBlue" />
+```gradle
+implementation 'net.skyscanner.backpack:backpack-compose:x.x.x' // not released yet
 ```
 
+All Compose components must to be used within `BpkTheme` scope:
+
 ```kotlin
-R.color.bpkSkyBlue
+BpkTheme {
+    BpkText("Hello from Compose!")
+}
 ```
-### Gradient
 
-The Backpack gradient component is available with the `BPKGradient` utility class. It accepts the direction of the gradient as an optional parameter.
+## Features
 
-```java
-BpkGradients.getPrimary(context);
-BpkGradients.getPrimary(context, GradientDrawable.Orientation.LEFT_RIGHT);
-```
+### Android View system
+
+You can find the list of the available components, as well as the code samples and the screenshots [here](https://backpack.github.io/components/text?platform=android).
+
+All design system tokens exist as public Android resources. Here's the list of the token types and the samples of its usage:
+
+* border radii – `@dimen/bpkBorderRadiusSm` (XML), `R.dimen.bpkBorderRadiusSm` (Java/Kotlin)
```

Comment:
can you add the link back to these as well and maybe add an e.g. before the example
also drawables are missing in both sections

Reply:
Drawables are not tokens, and they have their own page as components:
https://backpack.github.io/components/icon?platform=android

Reply:
we should add a compose version for this!

Reply:
Good point